crates/goose-server/src/routes/extension.rs

```diff
@@ -108,6 +108,59 @@ async fn add_extension(
         return Err(StatusCode::UNAUTHORIZED);
     }
 
+    // If this is a Stdio extension that uses npx, check for Node.js installation
+    if let ExtensionConfigRequest::Stdio { cmd, .. } = &request {
+        if cmd.ends_with("npx.cmd") || cmd.ends_with("npx") {
```

Comment:
this would be better if it was also conditional on windows wouldn't it?

Reply:
not sure what you mean by that. It will check if you have node on line 115 so if you do - it will skip this whole thing